packages/table/src/headers/header.tsx

```diff
@@ -435,7 +440,30 @@ export class Header extends React.Component<IInternalHeaderProps, IHeaderState>
         this.setState({ hasSelectionEnded: true });
     };
 
-    private isEntireCellTargetReorderable = (isSelected: boolean) => {
+    private isDragSelectableDisabled = (event: MouseEvent) => {
+        if (event.metaKey) {
+            // if the meta key is pressed, we want to forcefully ignore reordering
+            // interactions and prioritize drag-selection interactions (e.g. to make
+            // it possible to deselect a row).
```

Comment:
i don't think `meta` is the ideal key here for all platforms. `meta` means `cmd` on mac and `windows-key` on windows. typically i use what the hotkeys code calls `mod` which is `cmd` on mac and `ctrl` on windows. i think the `windows-key` accesses OS resources like the start menu so it's not usually used in a key combo.

Reply:
maybe holding _any_ modifier key should disable the drag?

Reply:
seems like this would likely conflict with other key combos

Reply:
Fixed by using `DragEvents.isAdditive(event)`